game-app/game-core/src/main/java/games/strategy/triplea/ui/menubar/help/HelpMenu.java

```diff
@@ -43,6 +44,26 @@ public static JMenu buildMenu(final UiContext uiContext, final GameData gameData
 
     menu.add(bugReportMenu).setMnemonic(KeyEvent.VK_B);
 
+    menu.addSeparator();
+
+    final JMenuItem hostingLink = new JMenuItem("User Guide");
+    hostingLink.setMnemonic(KeyEvent.VK_H);
+    hostingLink.addActionListener(
+        e -> SwingComponents.newOpenUrlConfirmationDialog(UrlConstants.USER_GUIDE));
+    menu.add(hostingLink);
+
+    final JMenuItem warClub = new JMenuItem("TripleA Forum");
+    warClub.setMnemonic(KeyEvent.VK_W);
+    warClub.addActionListener(
+        e -> SwingComponents.newOpenUrlConfirmationDialog(UrlConstants.TRIPLEA_FORUM));
+    menu.add(warClub);
+
+    final JMenuItem donateLink = new JMenuItem("Donate");
```

Comment:
A donate menu item in a 'help' menu seems a bit odd. I would expect this is an about menu or something. Let's discuss where it should belong, perhaps 'help' is the least worst place for it, IDK.

Reply:
It's the best place for the donate link for now. 

Creating a ```Help > About Triplea``` entry which opens a dialog makes a lot of sense for v3.0. 
The donate and maybe some other links could be moved there, the Triplea version and some artwork could be displayed, etc.

Reply:
After thinking about it, I'd agree it's a decent enough place. "Donate" is related to "help" (just it's the player helping TripleA vs the other way round).